src/gwt/src/org/rstudio/studio/client/workbench/views/jobs/view/JobsDisplayImpl.java

```diff
@@ -217,6 +217,11 @@ public void onSelected()
    {
    }
    
+   @Override
+   public void setFocus()
+   {
```

Comment:
Should this do something? (Set focus to the widget hosting the jobs list?)

Reply:
I think I made the correct update, can you take another look?